src/api/Controllers/BoreholeController.cs

```diff
@@ -142,6 +144,69 @@ private IQueryable<Borehole> GetBoreholesWithIncludes()
             .Include(b => b.UpdatedBy);
     }
 
+    /// <summary>
+    /// Exports the details of up to 100 boreholes as a CSV file. Filters the boreholes based on the provided list of IDs.
+    /// </summary>
+    /// <param name="ids">The list of IDs for the boreholes to be exported.</param>
+    /// <returns>A CSV file containing the details of up to 100 specified boreholes.</returns>
+    [HttpGet("export-csv")]
+    [Authorize(Policy = PolicyNames.Viewer)]
+    public async Task<IActionResult> DownloadCsv([FromQuery] IEnumerable<int> ids)
+    {
+        if (!ids.Any())
+        {
+            return BadRequest("The list of IDs must not be empty.");
+        }
+
+        var boreholes = await Context.Boreholes
+                                    .Where(borehole => ids.Contains(borehole.Id))
+                                    .Take(100)
```

Comment:
REQ: Dieses Limit in eine Konstante auslagern, das macht das Ganze klarer. Zudem kann in der Funktionsbeschreibung die Variable referenziert werden; bei Anpassungen wird so der Kommentar auch automatisch angepasst.